src/app/loadout/loadout-edit/LoadoutEditBucket.tsx

```diff
@@ -170,14 +170,27 @@ function ItemBucket({
 
   const showFashion = LockableBucketHashes.includes(bucketHash);
 
-  const handlePlaceholderClick = () => onClickPlaceholder({ bucket });
+  const handlePlaceholderClick = (equip: boolean) => onClickPlaceholder({ bucket, equip });
 
   // TODO: plumb through API from context??
   // TODO: expose a menu item for adding more items?
   // TODO: add-unequipped button?
   // T0DO: customize buttons in item popup?
   // TODO: draggable items?
 
+  const showAddUnequipped = equipped.length > 0 && unequipped.length < bucket.capacity - 1;
+
+  const addUnequipped = showAddUnequipped && (
+    <button
+      type="button"
```

Comment:
Getting a React error without a key here

```suggestion
      type="button"
      key="addbutton"
```